geonode_mapstore_client/client/js/epics/gnsave.js

```diff
@@ -67,17 +67,16 @@ const SaveAPI = {
         const description = metadata.description;
         const thumbnail = metadata.thumbnail;
         const body = {
-            name,
             data,
             attributes: [{
```

Comment:
I don't think we need the `attributes` property anymore. The `attributes` property were used to simulate the GeoStore (default backend of mapstore) request but now we are using directly the v2 so I think the body could be this, right?:


```js
const body = {
    'title': metadata.name,
    'abstract': metadata.description,
    'data': data,
    'thumbnail_url': metadata.thumbnail
};
```

So probably the mapping of attributes performed by the adpter could be completely remove

Reply:
Regarding this, i'm still usign that payload in order to let GeoNode update information regarding the map (are assigned directly to the resource), are not stored anymore as `MapStoreAttribute`. If we dont want to keep them as is, we should improve the payload of data with the information coming from the frontend in order to let the user be able to update the Title, abstract and thumb_url and then pop them from that payload.